sources/packages/web/src/services/StudentAppealService.ts

```diff
@@ -48,4 +50,15 @@ export class StudentAppealService {
       approvals,
     );
   }
+
+  /**
+   * Gets all pending student application exceptions.
```

Comment:
Is it still saying "application exceptions" on a `getPendingAppeals` method?

Reply:
I will update, thanks.